src/commands/validate-tx.cpp

```diff
@@ -51,23 +51,14 @@ console_result validate_tx::invoke(std::ostream& output,
 
     callback_state state(error, output);
 
-    auto on_done = [&state](const chain::point::indexes& indexes)
+    auto on_done = [&state](const code& error)
     {
-        if (indexes.empty())
-        {
-            state.output(BX_VALIDATE_TX_VALID);
-            return;
-        }
-
-        const auto unconfirmed = join(numbers_to_strings(indexes), ", ");
-        state.output(format(BX_VALIDATE_TX_UNCONFIRMED_INPUTS) % unconfirmed);
+        state.succeeded(error);
+        state.output(BX_VALIDATE_TX_VALID);
```

Comment:
This will produce a "valid" message regardless of the result code.

I recommend using the `BX_VALIDATE_TX_VALID` message in the case where there is no error but emitting the error message text in the case of an error. Ideally the error message text would be wrapped with a new format symbol that provides context for the error message and incorporates the message text, such as:

> "The transaction is invalid for the current state of the node: %1%."